pkg/mountutil/mountutil.go

```diff
@@ -82,30 +84,27 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore) (*Processed, error
 		if !filepath.IsAbs(dst) {
 			return nil, errors.Errorf("expected an absolute path, got %q", dst)
 		}
+		rawOpts := ""
 		if len(split) == 3 {
-			opts := strings.Split(split[2], ",")
-			for _, opt := range opts {
-				switch opt {
-				case "rw":
-					// NOP
-				case "ro":
-					ro = true
-				default:
-					logrus.Warnf("unsupported volume option %q", opt)
-				}
-			}
+			rawOpts = split[2]
+		}
+
+		// always call parseVolumeOptions for bind mount to allow the parser to add some default options
+		var err error
+		var specOpts []oci.SpecOpts
+		options, specOpts, err = parseVolumeOptions(res.Type, src, rawOpts)
+		if err != nil {
+			return nil, err
 		}
+		res.Opts = append(res.Opts, specOpts...)
 	default:
 		return nil, errors.Errorf("failed to parse %q", s)
 	}
 	res.Mount = specs.Mount{
-		Type:        "none",
+		Type:        "bind",
```

Comment:
Why

Reply:
Because this is how docker does. But TIL runc ignores this for bind mounts.

https://github.com/opencontainers/runc/blob/v1.0.0/libcontainer/specconv/spec_linux.go#L344-L352

```golang
	if flags&unix.MS_BIND != 0 {
		// Any "type" the user specified is meaningless (and ignored) for
		// bind-mounts -- so we set it to "bind" because rootfs_linux.go
		// (incorrectly) relies on this for some checks.
		device = "bind"
		if !filepath.IsAbs(source) {
			source = filepath.Join(cwd, m.Source)
		}
	}
```

Reverted this to `"none"`.